lib/gtk/v3_4/gtk_widgets_ext.nit

```diff
@@ -145,116 +145,104 @@ end
 
 extern class GtkColorSelectionDialog
 	super GtkWidget
-	new ( title : String, parent : GtkWindow ) is extern  import String.to_cstring `{
-		 return gtk_color_chooser_dialog_new( String_to_cstring( title ), parent );
+	new (title: String, parent: GtkWindow)  import String.to_cstring `{
+		 return gtk_color_chooser_dialog_new(String_to_cstring(title), parent);
 	`}
 
-	#fun color_selection :  is extern `{
-	#	return gtk_color_selection_dialog_get_color_selection( GTK_COLOR_SELECTION_DIALOG( recv ) );
-	#`}
+	# fun color_selection:  `{
```

Comment:
Same answer, they need to be moved to new modules. It is outside of the scope of this PR.

If you want, you can raise an issue for me to add versions to this module and to clean up string manipulations.